maintainers/team-list.nix

```diff
@@ -803,6 +803,13 @@ with lib.maintainers;
     enableFeatureFreezePing = true;
   };
 
+  neovim = {
+    members = [ ];
+    githubTeams = [ "neovim" ];
```

Comment:
I get the distinct impression (e.g. from https://github.com/NixOS/nixpkgs/commit/ed47d92c53deeabd4e55b5099e753ee731b8251e) that `githubTeams` is not intended to replace `members`, and that `members` should still contain all members of the team.

I'm pretty confused though: it looks like the unused(?) `vim` team contained @figsoda, @softinio and @teto, the old neovim maintainers were @manveru and @rvolosatovs , but https://github.com/orgs/NixOS/teams/neovim lists @khaneliman @mrcjkb @GaetanLepage and @PerchunPak . I had expected more overlap ;) - what is the most up-to-date list?

Reply:
ah I had missed #385792, sorry about the noise